src/coresyftools/coresyftools/packager.py

```diff
@@ -110,9 +110,14 @@ def pack_tool(self):
 @click.option('--scihub_pass')
 def pack_tool(tool_dir, target_dir, scihub_user, scihub_pass):
     click.echo('Packaging {} to {}..'.format(tool_dir, target_dir))
-    packager = Packager(tool_dir, target_dir, (scihub_user, scihub_pass))
-    packager.pack_tool()
-    click.echo('Packaging finished.')
+    try:
+        packager = Packager(tool_dir, target_dir, (scihub_user, scihub_pass))
+        packager.pack_tool()
+    except ToolErrorsException as tool_errors:
```

Comment:
This looks fine and improves the tool tester logs. However, I have a question:
What if we get a different Exception, such as  "ToolDirectoryNotFoundException", "ManifestFileNotFound" or other? The idea is do not treat this exceptions and let them interrupt the execution?

Reply:
We will accept this PR. We assume that the other Exceptions are self-explanatory and do not need to be treated before interrupt the execution of the packager. @91nunocosta do you agree?